gcp-function-http/src/main/java/io/micronaut/gcp/function/http/GoogleFunctionHttpRequest.java

```diff
@@ -185,61 +189,13 @@ public MutableConvertibleValues<Object> getAttributes() {
     @NonNull
     @Override
     public Optional<B> getBody() {
-        return (Optional<B>) getBody(Argument.OBJECT_ARGUMENT);
+        return this.body.get();
     }
 
     @NonNull
     @Override
     public <T> Optional<T> getBody(@NonNull Argument<T> arg) {
-        if (arg != null) {
```

Comment:
I changed this to use the [BodyBuilder API](https://github.com/micronaut-projects/micronaut-servlet/pull/451) introduced here: